pkg/endpoints/pipeline_test.go

```diff
@@ -175,6 +199,25 @@ func TestPipeline(t *testing.T) {
 			t.Fatalf("Response object %v did not equal expected %v\n", responsePipeline, pipeline)
 		}
 	}
+
+	ns1Pipelines := pipelines[:2]
```

Comment:
Instead of this, it would be more explicit if there were two groups/slices of objects rather than the sub-slice since "potentially" someone could add another ns1 pipeline (or similar) at different segments of the the declaration. Same situation above, but in any case that would break the tests, which it would be self diagnosing. At your discretion if you want to make the change.